tests/dummy/app/index/template.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="ui segment basic">
+  <h3>Components</h3>
+
+  {{#nrg-home-cards as |view|}}
+    {{view.home-card route="/view-components/nrg-error-pages" label="Error Pages" icon="warning sign"}}
+    {{view.home-card route="/view-components/nrg-form-container" label="Form Container" icon="tasks"}}
```

Comment:
How about we rename the form container route to just "Forms" and can you change this home card to match?

Reply:
Also maybe the `edit` icon?